src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import './Nav.scss';
+import { Link } from 'react-router-dom';
+import './NavSmall';
+import NavSmall from './NavSmall';
+
+// true false
+const Navv = () => {
+  const [show, setShow] = useState(true);
+  const [middleOn, setMiddleOn] = useState(false);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      window.scrollY > 1 ? setShow(false) : setShow(true);
+    });
+    return () => {
+      window.removeEventListener('scroll');
+    };
+  }, []);
+
+  const handleSearchOpen = () => {
+    middleOn ? setMiddleOn(false) : setMiddleOn(true);
+  };
+
+  if (show) {
+    return (
+      <div className="Navv">
+        <div className="header">
+          <div className="inner">
+            <div className="first">
+              <h1>
+                <Link to>
+                  <img src="./images/Nav/NavLogo.jpg" alt="테라로사" />
+                </Link>
+              </h1>
+            </div>
```

Comment:
```suggestion
            <Link to="/" className="first">
              <img src="./images/Nav/NavLogo.jpg" alt="테라로사" />
            </Link>
```
불필요하게 deps가 너무 깊습니다
h1 태그를 사용해서 해당 로고를 표현하고 싶다면
Link 대신 h1을 쓰고 onclick을 걸어주면 될 것 같습니다

className 역시 first 보다는 logo가 더 직관적인 표현이 될 것 같습니다